config/initializers/rack/attack.rb

```diff
@@ -22,4 +22,10 @@ class Rack::Attack
       request.env["HTTP_FASTLY_CLIENT_IP"].to_s
     end
   end
+
+  throttle("message_throttle", limit: 1, period: 1) do |request|
```

Comment:
Maybe we bump this up to 2 messages per second? Sometimes if I'm pasting a link or doing short burst messages, I legitimately exceed 1 message/second. I imagine that doesn't happen with most people though.

Reply:
I started with 2 then thought is that really possible and lowered it to 1. I could see in a copy paste scenario maybe so I will up it